### PR TITLE
Updated Auth Flow

### PR DESCRIPTION
**Pull request by shreypandey**

### CODE REVIEW

**Comment by shreypandey** [comment]
@microsoft-github-policy-service agree [company="Microsoft"]

File: studio/src/pages/home-page.tsx [hunk 1]
```diff
@@ -50,11 +50,11 @@ export const HomePage: React.FunctionComponent = () => {
     React.useEffect(() => {
         if (account && inProgress === "none") {
             instance.acquireTokenSilent({
-                scopes: [import.meta.env.VITE_REACT_APP_ADD_APP_SCOPE_URI || ''],
+                scopes: ["user.read"],
```
Comment (sameersegal) [inline studio/src/pages/home-page.tsx:53]: Roll back this change. This is not the problem

**Review by sameersegal — APPROVED** [review]
Merging this for now as we need it quickly but we need to revert the change on the scope

File: server/app/main.py [hunk 2]
```diff
@@ -79,31 +79,36 @@ def get_db():
...
-        unverified_token = jwt.get_unverified_header(token.replace("Bearer ", ""))
-        return jwt.decode(
-            token.replace("Bearer ", ""),
-            issuer=ISSUER,
-            audience=AAD_APP_CLIENT_ID,
-            key=key,
-            algorithms=[unverified_token["alg"]],
-            options={"verify_signature": False},
-        )
+        id_token = token.split(" ")[1]
```
Comment (sameersegal) [inline server/app/main.py:110]: The previous way for extracting the token was better. This makes a hard assumption that the token is going to  have one space in it

**Pull request merged**